api/base/urls.py

```diff
@@ -16,31 +16,31 @@
             [
                 url(r'^$', views.root, name='root'),
                 url(r'^status/', views.status_check, name='status_check'),
-                url(r'^actions/', include('api.actions.urls', namespace='actions')),
-                url(r'^addons/', include('api.addons.urls', namespace='addons')),
-                url(r'^applications/', include('api.applications.urls', namespace='applications')),
-                url(r'^citations/', include('api.citations.urls', namespace='citations')),
-                url(r'^collections/', include('api.collections.urls', namespace='collections')),
-                url(r'^comments/', include('api.comments.urls', namespace='comments')),
+                url(r'^actions/', include(('api.actions.urls', 'actions'), namespace='actions')),
```

Comment:
I'm a bit confused by this change. 

According to the docs, `include(module, namespace=None, app_name=None)` is valid usage.

Passing a 3-tuple is deprecated, but we weren't doing that in the first place.

Passing a 2-tuple is valid, i.e. `include((pattern_list, app_namespace), namespace=None)`, but if I'm understanding correctly, the 2nd item of the tuple is supposed to be the app name. `actions`, `addons`, `applications`, etc. aren't proper Django apps (they're not in `INSTALLED_APPS`), so I'm not sure it's correct to pass them as app names.

Reply:
`include(module, namespace=None, app_name=None)` is a valid signature in Django 1.11, but won't be in 2.0 (see the 2.0 docs [here](https://docs.djangoproject.com/en/2.0/ref/urls/#include)). 

I do agree with your last point re: `actions`, `addons`, etc. being improper arguments for `app_namespace` and I'm looking into that now.

Reply:
After reading the Django 2.0 docs further, I think the recommended approach is to specify `app_name` (which should be `osf`) in the included modules and continue using the `include(module, namespace=None)` signature we have now. 

Note that the difference in Django 2.0 is that `app_name` is required (i.e. it needs to be set in the included module and can no longer be `None`).

Reply:
Ah ok, thanks for the clarification.